project/yuanhenglee/README.rst

```diff
@@ -0,0 +1,179 @@
+# BasiCPP Pitch: A C++ implementation for AMT(Automatic Music Transcription)
+
+BasiCPP Pitch is a instrument-agnostic and polyphonic capable AMT(Automatic
+Music Transcription) library written in C++.
+
+## Basic Information
+
+BasiCPP Pitch is an instrument-agnostic and polyphonic-capable AMT (Automatic
+Music Transcription) library written in C++.
+
+- github repository:[repo](https://github.com/yuanhenglee/basicpp-pitch)
+
+Provide any compatible audio file, the library will generate a MIDI file with
```

Comment:
It won't be hard to make beautiful slides using the waveform analysis and notes.